Vates/VatesAPI/src/SaveMDWorkspaceToVTK.cpp

```diff
@@ -0,0 +1,106 @@
+#include "MantidVatesAPI/SaveMDWorkspaceToVTK.h"
+#include "MantidAPI/WorkspaceProperty.h"
+#include "MantidAPI/IMDWorkspace.h"
+#include "MantidAPI/IMDHistoWorkspace.h"
+#include "MantidAPI/IMDEventWorkspace.h"
+#include "MantidAPI/FileProperty.h"
+#include "MantidKernel/ListValidator.h"
+
+#include "MantidVatesAPI/Normalization.h"
+#include "MantidVatesAPI/SaveMDWorkspaceToVTKImpl.h"
+
+#include "MantidKernel/BoundedValidator.h"
+
+namespace Mantid {
+namespace VATES{
+
+  DECLARE_ALGORITHM(SaveMDWorkspaceToVTK)
+
+  SaveMDWorkspaceToVTK::SaveMDWorkspaceToVTK() : pimpl(new SaveMDWorkspaceToVTKImpl) {}
```

Comment:
I haven't seen pimpl done quite like this before. Maybe we could talk about pros-cons.